build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.1'
```

Comment:
Android gradle plugin version should stay at 3.5.1 to avoid build failure on Douglas computers